### ISSUE TITLE
Text colour issue in dark theme

### ISSUE THREAD

**Issue by sith-on-mars** [GabePoel/KvLibadwaita#6]
Some texts in the dark mode are using the wrong colour. Not sure why it looks different from the images from the readme file. Here are a few examples:

![image](https://user-images.githubusercontent.com/37974764/183005718-9c80a0f0-950e-4f5c-a8f8-ebef5923a65b.png)

![image](https://user-images.githubusercontent.com/37974764/183006314-aecac51b-4ae7-4a0d-b3c3-70f3f4a16211.png)

**Comment by ruvaldak** [comment]
Yep, same issue here. Actually, quite a bit of issues, overall looks very poor. Rolled back to 152c204 and it looks much better

**Comment by GabePoel** [comment]
Hmm... That's odd. I'm having difficulty replicating the issue. But, I'll poke at it and see what I can do. Does setting the QT style to `kvantum-dark` instead of `kvantum` seem to help? (You can try this with the `QT_STYLE_OVERRIDE=kvantum-dark` environment variable.)

It also looks like one of the thumbnail colors is a little funky... I'll see if I can fix that as well.

**Comment by sith-on-mars** [comment]
> Hmm... That's odd. I'm having difficulty replicating the issue. But, I'll poke at it and see what I can do. Does setting the QT style to `kvantum-dark` instead of `kvantum` seem to help? (You can try this with the `QT_STYLE_OVERRIDE=kvantum-dark` environment variable.)
> 
> It also looks like one of the thumbnail colors is a little funky... I'll see if I can fix that as well.

Changing Qt style to `kvantum-dark` doesn't change anything on my end.

**Comment by ruvaldak** [comment]
Nor mine. No change.

**Comment by alissonlauffer** [comment]
Not only that issue, but I'm also having issues with Qt lists having a "teal" background when the window is inactive or moving:
![Screenshot from 2022-08-05 16-43-30](https://user-images.githubusercontent.com/29029632/183150199-dee3c2e8-3559-4141-ae35-819180d5deb9.png)
![Screenshot from 2022-08-05 16-48-50](https://user-images.githubusercontent.com/29029632/183150340-93f4c3b8-d158-4727-b0a8-34dd4120805f.png)

**Comment by GabePoel** [comment]
Alright. I've reverted the theme to its previous version until I get this fixed.
